src/SharpLearning.Neural.Test/Learners/ClassificationNeuralNetLearnerTest.cs

```diff
@@ -92,5 +119,15 @@ public void ClassificationNeuralNetLearner_Constructor_Throw_On_Wrong_OutputLaye
 
             return (observations, targets);
         }
+
+        (double[][] observations, double[] targets) CreateArrayData(
+            int numberOfObservations, int numberOfFeatures, int numberOfClasses, Random random)
+        {
+            var observations = Enumerable.Range(0, numberOfObservations).Select(i => Enumerable.Range(0, numberOfFeatures).Select(ii => random.NextDouble()).ToArray()).ToArray();
+            var targets = Enumerable.Range(0, numberOfObservations).Select(i => (double)random.Next(0, numberOfClasses)).ToArray();
```

Comment:
@jameschch nit: can you reduce the line lengths a bit here. Just break the lines of the linq statements a bit.

Reply:
Done